app/src/main/java/com/usepace/android/messagingcenter/clients/connection_client/ClientInterface.java

```diff
@@ -20,6 +20,8 @@
 abstract class ClientInterface {
 
     abstract public void connect(Context context, ConnectionRequest connectionRequest, ConnectionInterface connectionInterface);
+    abstract public void reConnect();
```

Comment:
it shouldn't be accessible in the app level

Reply:
It is already not accessible to outside app, and I have just added 2 new methods to it.